### PR TITLE
GitHub discussions isn't used anymore, dead link

### PR DESCRIPTION
**Pull request by cool-RR**
Signed-off-by: Ram Rachum <ram@rachum.com>

## Why are these changes needed?

Dead link

## Related issue number

No issue

## Checks

- [V] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [V] I've run `scripts/format.sh` to lint the changes in this PR.
- [V] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [V] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -6,9 +6,8 @@ Reporting bugs and asking questions
 
 You can post questions or issues or feedback through the following channels:
 
-1. `Github Discussions`_: For discussions about development, questions about usage, and feature requests.
-2. `GitHub Issues`_: For bug reports and feature requests.
-3. `StackOverflow`_
+1. `GitHub Issues`_: For bug reports and feature requests.
```
Comment (jjyao) [inline CONTRIBUTING.rst:9]: Could you replace Github Discussions with Discourse Forum `For discussions about development and questions about usage.`?

Also line 21 needs to be updated as well.

**Review by jjyao — COMMENTED** [review]
Thanks!!!

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ Reporting bugs and asking questions
 
 You can post questions or issues or feedback through the following channels:
 
-1. `Github Discussions`_: For discussions about development, questions about usage, and feature requests.
+1. `Discourse forum`_: For discussions about development, questions about usage, and feature requests.
```
Comment (jjyao) [inline CONTRIBUTING.rst:9]: Let's remove `, and feature requests`. Feature requests goes to github issues.

**Review by jjyao — APPROVED** [review]

File: CONTRIBUTING.rst (hunk 2 above)
Comment (cool-RR) [reply CONTRIBUTING.rst]: Done.

**Pull request merged**